app/main/templates/workbench.html

```diff
@@ -5,147 +5,157 @@
-                        {% for task in batch_form.task_name.choices %}
-                            <li><input type="radio" name="task_name" value={{task}}> {{task}}</li>
-                        {% endfor %}
-                    </ul>
-                    </br>
-                    <label for="task_variable">optional command arguments: </label><input type="string" name="task_variable"/>
-                    </br>
-                    <p>For example "--measured_slice_width=3" for the SNR task</p>
-                    </br>
-                    {{ batch_form.submit }}
-                </div>
-            </div>
-            <div class="col-md-8 col-lg-8">
-                <div class="tab-content">
-                    <h4 class="text-center pt-2">Select images</h4></br>
-                    {% for study_device in study_device_list %}
-                    <div class="accordion" id="accordionExample">
-                        <div class="accordion-item">
-                            <h2 class="accordion-header" id="heading{{ loop.index }}">
-                                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse{{ loop.index }}" aria-expanded="true" aria-controls="collapse{{ loop.index }}">{{ study_device.device.institution}} , {{ study_device.device.manufacturer }}, {{ study_device.device.device_model }}</button>
-                            </h2>
+    <div class="hazen-dotted-horizontal-line mb-3"></div>
+
+    <form method="POST" action={{ url_for('main.workbench') }}>
+        <div class="container-fluid">
+
+
+            <!-- Task list Scrollspy -->
+            <div class="row pt-5">
+                <div class="col-md-3 col-lg-3 pb-3">
+                    <div id="scrollspy1" style="background-color: #D2D7DB" class="sticky-top">
+                        <div class="p-3 pb-4">
+                            <h4 class="text-center" style="font-family: Monda, sans-serif; color: #000">Task Selector</h4>
+                            <ul class="nav flex-column nav-pills menu-sidebar" id="tasks">
+                                {% for task in batch_form.task_name.choices %}
+                                    <li><input type="radio" name="task_name" value={{task}}> {{task}}</li>
+                                {% endfor %}
+                            </ul>
+
+                            <label class="pt-3" for="task_variable" style="font-weight: bold">
+                                Enter optional hazen arguments:
+                            </label>
+                            <div class="input-group mb-3">
+                                <span class="input-group-text" id="task_variable"></span>
+                                <input type="text" class="form-control" placeholder="--measured_slice_width=3"
+                                       aria-label="Username" aria-describedby="task_variable">
+                            </div>
```

Comment:
I think we should look into this further. passing task variable this way is not currently functional